lib/core/models/Config.ts

```diff
@@ -4,6 +4,7 @@
 export default interface Config {
   batchingIntervalInSeconds: number;
   blockchainServiceUri: string;
+  blockchainFeeMarkupFactor: number;
```

Comment:
Nit:`transactionFeeMarkup...`.

Optional: consider `...FeeMarkupPercentage` as "percentage" is be more easy to understand and there is less interpretation than "factor", obviously you'll need to divide by 100... so I understand if push back.

Reply:
Will change this to 'transactionFeeMarkupPercentage'. Also change the 'feePaid' returned by the blockchain service to 'transactionFeePaid.'